flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/api/TableEnvironmentTest.java

```diff
@@ -127,6 +130,25 @@ void testTableFromDescriptor() {
         assertThat(tEnv.getCatalogManager().listTables()).isEmpty();
     }
 
+    @Test
+    void testGetQueryOperationDefaultJobName() {
+        final TableEnvironmentMock tEnv = TableEnvironmentMock.getStreamingInstance();
+        String sql1 = "select * from t";
+        String sql2 = "";
+
+        QueryOperation mockOperation1 = new QuerySqlOperationMock(sql1);
+        QueryOperation mockOperation2 = new QuerySqlOperationMock(sql2);
+        QueryOperation mockOperation3 = new QueryOperationMock();
+
+        String jobName1 = tEnv.getDefaultJobName(mockOperation1);
```

Comment:
The test seems only verifying `TableEnvironmentMock#getDefaultJobName` method. Is it possible to have tests that verify the whole process (something we can call integration tests).